consensus/core/src/metrics.rs

```diff
@@ -157,8 +157,8 @@ pub(crate) struct NodeMetrics {
     pub(crate) block_manager_missing_ancestors_by_authority: IntCounterVec,
     pub(crate) threshold_clock_round: IntGauge,
     pub(crate) subscriber_connection_attempts: IntCounterVec,
-    pub(crate) subscriber_connections: IntGaugeVec,
-    pub(crate) subscribed_peers: IntGaugeVec,
+    pub(crate) subscribed_to: IntGaugeVec,
+    pub(crate) subscribed_by: IntGaugeVec,
```

Comment:
I like that we are renaming those - indeed quite hard to always remember. I like the new approach. Just leaving a couple of other ideas just for the sake of brainstorming:

```
| subscribed_to         | subscribed_by          |
|-----------------------|------------------------|
| subscribed_to_peer    | subscribed_to_me        |
| outbound_subscriptions| inbound_subscriptions   |
| streaming_from        | streaming_to            |
| consuming_from        | publishing_to           |
```

Reply:
Sorry didn't see this before merging. Yes we can iterate on the naming.